Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -191,3 +191,24 @@
       - id: RubberStampHos
       - id: SecurityTechFabCircuitboard
       - id: JetpackSecurityFilled
+
+- type: entity
+  id: LockerBlueShieldFilled
+  suffix: Filled
+  parent: LockerBlueShield
+  components:
+  - type: StorageFill
+    contents:
+      - id: WeaponDisabler
+      - id: PinpointerNuclear
+      - id: ClothingUniformJumpskirtBlueShield
+      - id: ClothingUniformJumpsuitBlueShield
+      - id: ClothingEyesGlassesSunglasses
+      - id: ClothingHeadsetCommand
+      - id: ClothingHandsGlovesCombat
+      - id: ClothingShoesBlueShieldShoes
+      - id: ClothingOuterBlueShieldVest
+      - id: ClothingHeadBeretBlueShield
+      - id: MedkitCombatFilled
+      - id: MedkitFilled
```

Comment:
this is a lot of healing to just be giving out for free

Reply:
well, he has to treat, 2 first aid kits